discord/ui/select.py

```diff
@@ -141,13 +162,22 @@ def __init__(
         disabled: bool = False,
         row: int | None = None,
         id: int | None = None,
+        label: str | None = None,
+        description: str | None = None,
+        required: str | None = None,
```

Comment:
```suggestion
        required: bool | None = None,
```
I think ? Or did I miss smth ?

Reply:
required="True"